intrahost.py

```diff
@@ -474,6 +474,7 @@ def merge_to_vcf(
         outf.write('##fileformat=VCFv4.1\n')
         outf.write('##FORMAT=<ID=GT,Number=1,Type=String,Description="Genotype">\n')
         outf.write('##FORMAT=<ID=AF,Number=A,Type=Float,Description="Allele Frequency">\n')
+        outf.write('##FORMAT=<ID=DP,Number=1,Type=Integer,Description="Read depth at position">\n')
```

Comment:
Description should be "Read Depth" in keeping with [VCF fixed vocabulary](http://samtools.github.io/hts-specs/VCFv4.1.pdf).